### ISSUE TITLE
Bluetooth: IPSP Sample Doc no match for new path

### ISSUE THREAD

**Issue by lgl88911** [zephyrproject-rtos/zephyr#9727]
https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/bluetooth/ipsp show:
To test IPSP please take a look at samples/net/README
but I can't find the file

**Comment by Vudentz** [comment]
Yep, that seems outdated, I will update with the instruction how to use the echo-client which is the only thing we are really using from net-tools.

**Comment by lgl88911** [comment]
@Vudentz  Other question about the doc: Where is bluetooth_6lowpan file, I can't find it in zephyr tree. Thanks!

**Comment by Vudentz** [comment]
That is on Linux side, that session describes how to setup the Linux box to talk to zephyr.
